packages/app-store/pipedrive-crm/api/_postAdd.ts

```diff
@@ -0,0 +1,61 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { stringify } from "querystring";
+
+import { HttpError } from "@calcom/lib/http-error";
+import logger from "@calcom/lib/logger";
+import { defaultResponder } from "@calcom/lib/server";
+import prisma from "@calcom/prisma";
+
+import checkSession from "../../_utils/auth";
+import getInstalledAppPath from "../../_utils/getInstalledAppPath";
+import appConfig from "../../pipedrive-crm/config.json";
+
+export async function getHandler(req: NextApiRequest, res: NextApiResponse) {
+  const session = checkSession(req);
+  if (!session.user) {
+    throw new HttpError({ statusCode: 401, message: "Unauthorized" });
+  }
+  //this need to be accessed by admin only. need to confirm
+  const { client_id, client_secret }: { client_id: string; client_secret: string } = req.body;
+  if (!client_id || !client_secret)
+    throw new HttpError({ statusCode: 400, message: "client_id and client_secret are required" });
+  //Not sure if it is valid or not. right now any user can change the pipe drive credentials.
+  const data = {
+    slug: appConfig.slug,
+    keys: { client_id, client_secret },
+    enabled: true,
+    categories: appConfig["categories"],
+    dirName: appConfig["dirName"],
+  } as any;
```

Comment:
```suggestion
const data: Prisma.AppCreateInput = {
    slug: appConfig.slug,
    keys: { client_id, client_secret },
    enabled: true,
    categories: appConfig["categories"] as unknown as 
    Prisma.AppCreatecategoriesInput | undefined,
    dirName: appConfig["dirName"],
  };
```
it's better to use proper type instead any.